generate_svn_version.sh

```diff
@@ -19,6 +19,7 @@ elif [ -d .git ]; then
     fi
     URL=`git config --get remote.$remote.url`
     DATE=`git log -n1 --pretty="format:%ct"`
+    SERVER_VERSION=`git describe --abbrev=0 --match server_release/*` 
```

Comment:
I think this should also be added to https://github.com/BOINC/boinc/blob/61d50f69ce140ce05eb4908e58b1db2d7eb12074/generate_svn_version.sh#L30, right at the end of `SVN_VERSION`, e.g. like this: `($HOST:$PWD [$BRANCH]) [Server-Release: $SERVER_VERSION]`

Reply:
I have made this change